pkg/app/gen/html.go

```diff
@@ -1311,6 +1312,11 @@ var attrs = map[string]attr{
 		Type: "string",
 		Doc:  "refers to a datalist element that contains pre-defined options for an input element.",
 	},
+	"loading": {
```

Comment:
Google is a reference but I want to keep things based on standards.

Reply:
Well I looked at the implementation information and it is now implemented on firefox too and they work on safari so it will be standard soon. Anyway, I remove it here and leave it in our own fork.